src/main/java/org/itsallcode/openfasttrace/importer/markdown/MarkdownImporter.java

```diff
@@ -38,87 +41,112 @@
+        transition(SPEC_ITEM  , COMMENT    , MdPattern.COMMENT    , this::beginComment                         ),
+        transition(SPEC_ITEM  , COVERS     , MdPattern.COVERS     , () -> {}                                   ),
+        transition(SPEC_ITEM  , DEPENDS    , MdPattern.DEPENDS    , () -> {}                                   ),
+        transition(SPEC_ITEM  , NEEDS      , MdPattern.NEEDS_INT  , this::addNeeds                             ),
+        transition(SPEC_ITEM  , NEEDS      , MdPattern.NEEDS      , () -> {}                                   ),
+        transition(SPEC_ITEM  , DESCRIPTION, MdPattern.DESCRIPTION, this::beginDescription                     ),
+        transition(SPEC_ITEM  , DESCRIPTION, MdPattern.NOT_EMPTY  , this::beginDescription                     ),
+        transition(SPEC_ITEM  , TAGS       , MdPattern.TAGS       , () -> {}                                   ),
     
-        transition(State.DESCRIPTION, State.SPEC_ITEM  , MdPattern.ID         , () -> {endDescription(); beginItem();}     ),
-        transition(State.DESCRIPTION, State.TITLE      , MdPattern.TITLE      , () -> {endDescription(); endItem();}       ),
-        transition(State.DESCRIPTION, State.RATIONALE  , MdPattern.RATIONALE  , () -> {endDescription(); beginRationale();}),
-        transition(State.DESCRIPTION, State.COMMENT    , MdPattern.COMMENT    , () -> {endDescription(); beginComment();}  ),
-        transition(State.DESCRIPTION, State.COVERS     , MdPattern.COVERS     , this::endDescription                       ),
-        transition(State.DESCRIPTION, State.DEPENDS    , MdPattern.DEPENDS    , this::endDescription                       ),
-        transition(State.DESCRIPTION, State.NEEDS      , MdPattern.NEEDS_INT  , () -> {endDescription(); addNeeds();}      ),
-        transition(State.DESCRIPTION, State.NEEDS      , MdPattern.NEEDS      , () -> endDescription()                     ),
-        transition(State.DESCRIPTION, State.DESCRIPTION, MdPattern.EVERYTHING , this::appendDescription                    ),
+        transition(DESCRIPTION, SPEC_ITEM  , MdPattern.ID         , () -> {endDescription(); beginItem();}     ),
+        transition(DESCRIPTION, TITLE      , MdPattern.TITLE      , () -> {endDescription(); endItem();}       ),
+        transition(DESCRIPTION, RATIONALE  , MdPattern.RATIONALE  , () -> {endDescription(); beginRationale();}),
+        transition(DESCRIPTION, COMMENT    , MdPattern.COMMENT    , () -> {endDescription(); beginComment();}  ),
+        transition(DESCRIPTION, COVERS     , MdPattern.COVERS     , this::endDescription                       ),
+        transition(DESCRIPTION, DEPENDS    , MdPattern.DEPENDS    , this::endDescription                       ),
+        transition(DESCRIPTION, NEEDS      , MdPattern.NEEDS_INT  , () -> {endDescription(); addNeeds();}      ),
+        transition(DESCRIPTION, NEEDS      , MdPattern.NEEDS      , () -> endDescription()                     ),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)